Bottom: 9faaf2e1cfb04b1bbfe4406c5671cc59c19ff513
Top:    080887f6baf4cf372496dbb4cc14722cf07f83f1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-05-08 13:40:07 +0100

Refresh of fix-crash-hovering-over-unconn

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 4922cb4..346a6db 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -1989,7 +1989,7 @@ describe_location (LocationType X, LocationType Y)
       GetLayerNumber (PCB->Data, (LayerTypePtr) ptr1) >= max_layer)
     return NULL;
 
-  elename = UNKNOWN (NAMEONPCB_NAME ((ElementTypePtr) ptr1));
+  elename = "DUMMY"; // UNKNOWN (NAMEONPCB_NAME ((ElementTypePtr) ptr1));
   pinname = ConnectionName (type, ptr1, ptr2);
 
   if (pinname == NULL)
